src/main/java/net/sf/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -1188,9 +1185,9 @@ private void italicizeOrBold(XTextCursor position, boolean italicize, int start,
     }
 
     private void removeReferenceMark(String name) throws NoSuchElementException, WrappedTargetException {
-        XReferenceMarksSupplier xSupplier = UnoRuntime.queryInterface(XReferenceMarksSupplier.class, xCurrentComponent);
-        if (xSupplier.getReferenceMarks().hasByName(name)) {
-            Object o = xSupplier.getReferenceMarks().getByName(name);
+        XNameAccess nameAccess = getReferenceMarks();
+        if (nameAccess.hasByName(name)) {
+            Object o = nameAccess.getByName(name);
```

Comment:
For sure it is possible. I just left it as is (and have no real idea what it does since I didn't consider that part of the code). While it looks like added code, it is just a consequence of the diff algorithm not really working.